### PR TITLE
CSS Issues 

### PR DESCRIPTION
**Pull request by hendriku**
@MalteBartels did 

- Fix several CSS bugs
- Improve chat window resposiveness
- Apply design to messages

### CODE REVIEW

File: src/main/web/app/components/Calendar.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ export default function Calendar(props) {
   return (
     <container>
       <div className="calendar">
-        <TimeView start={8} end={18} />
+        <TimeView start={7} end={18} />
```
Comment (hendriku) [inline src/main/web/app/components/Calendar.js]: @MalteBartels Is it correct, that we start from 7am now?

File: src/main/web/app/scss/messages.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
+$lightgray: #eaeaef;
+$primarylight: #ba3838;
+
+// Based on a pen by Jon Adams: https://codepen.io/pointlessjon/pen/hCEHG?page=1&
```
Comment (hendriku) [inline src/main/web/app/scss/messages.scss]: 👍

**Review by hendriku — COMMENTED** [review]
Great, ready to merge if @MalteBartels answers the last question.

File: src/main/web/app/components/Calendar.js (hunk 1 above)
Comment (MalteBartels) [reply src/main/web/app/components/Calendar.js]: No it's not and I changed it. I don't know why it appears here

**Pull request merged**